rgw/v2/tests/s3_swift/configs/test_sts_aswi_verify_role_policy_allow_actions.yaml

```diff
@@ -0,0 +1,59 @@
+# polarion test case id: CEPH-83593390
+# test scripts : test_sts_aswi.py
+config:
+     bucket_count: 2
+     objects_count: 50
+     objects_size_range:
+          min: 5
+          max: 15
+     encryption_keys: kms
+     lifecycle_conf:
+        - ID: LC_Rule_1
+          Filter:
+            Prefix: key2
+          Status: Enabled
+          Expiration:
+            Days: 20
+     test_ops:
+          create_bucket: true
+          create_object: true
+          endpoint: kafka
+          ack_type: broker
+          verify_policy: role_policy
+          bucket_tags: [
+              {
+                  "Key": "product",
+                  "Value": "ceph"
+              }
+          ]
+     sts:
+          policy_document:
+                "Version": "2012-10-17"
+                "Statement": [
+                    {
+                         "Effect": "Allow",
+                         "Principal": {
+                           "Federated": ["arn:aws:iam:::oidc-provider/ip_addr:8180/realms/master"]
+                         },
+                         "Action": ["sts:AssumeRoleWithWebIdentity"],
+                    }
+                ]
+          role_policy:
+               "Version": "2012-10-17"
+               "Statement":
+                [
+#                    {
```

Comment:
could you please remove the commented data, if its not required?

Reply:
@anrao19 
I have added them as a placeholder for future actually if the bz is verified
I will remove it if you want

Reply:
@anrao19 
done, pls review again